app/controllers/posts_controller.rb

```diff
@@ -17,4 +17,23 @@ def show
   def post_params
     params.permit(:user_id, :id)
   end
+
+  def new
+    @id = current_user.id
+    render 'new'
+  end
+
+  def create
+    title = params[:post]['title']
+    text = params[:post]['text']
```

Comment:
- [ ] [OPTIONAL] It would have been better if you had used strong parameters for this. You can read more about it [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides)